packages/network/bin/tracker.js

```diff
@@ -23,6 +24,10 @@ program
 const id = program.opts().id || `TR${program.opts().port}`
 const name = program.opts().trackerName || id
 const logger = new Logger(module)
+const listenConfig = program.opts().unixSocket ? program.opts().unixSocket : {
+    hostname: program.opts().ip,
+    port: program.opts().port
```

Comment:
does this need the `Number.parseInt(program.opts().port, 10)` that was there before?